### PR TITLE
Update ruby 2.6

### PR DESCRIPTION
**Pull request by nk-ty**
I caught the following error.
```
docker-compose -f docker-compose.dev.yml up --build -d
~~~
ERROR:  Error installing bucky-core:
 	The last version of public_suffix (>= 2.0.2, < 6.0) to support your Ruby & RubyGems was 4.0.7. Try installing it with `gem install public_suffix -v 4.0.7` and then running the current command again
 	public_suffix requires Ruby version >= 2.6. The current ruby version is 2.5.8.224.
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ RUN apk update && \
     build-base \
     curl-dev \
     git \
-    iproute2-minimal \
```
Comment (nk-ty) [inline Dockerfile:12]:
```bash
#5 2.072 ERROR: unable to select packages:
#5 2.072   iproute2-minimal (no such package):
#5 2.080     required by: world[iproute2-minimal]
```

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.5.8-alpine
+FROM ruby:2.6.0-alpine
```
Comment (hikimochi) [inline Dockerfile:1]: 2.6.10-alpine makes some errors?

Comment (nk-ty) [reply Dockerfile]: There is no error. So updated to 2.6.10 !

**Review by hikimochi — APPROVED** [review]
LGTM

**Pull request merged**